userguide/sharing.rst

```diff
@@ -1006,131 +1006,136 @@ for more details.
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | commit              | Track the amount of data written to a file and synchronizes it to disk when a specified amount accumulates.                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | crossrename         | Allow server side rename operations even if source and target are on different physical devices. Required for the recycle bin   |
+   |                     | to work across dataset boundaries. Automatically added when :guilabel:`Export Recycle Bin` is enabled.                          |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | default_quota       | **Deprecated: use the ixnas module instead.** Store the default quotas that are reported to a Windows client in the quota       |
+   |                     | record of a user.                                                                                                               |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | dirsort             | Sort directory entries alphabetically before sending them to the client.                                                        |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | expand_msdfs        | Enable support for Microsoft Distributed File System (DFS).                                                                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | extd_audit          | Send :guilabel:`audit` logs to both syslog and the Samba log files.                                                             |
```

Comment:
I don't think guilabel is correct here.  I think it does not need any markup at all, these are just audit logs.